docs/meson.build

```diff
@@ -71,10 +71,21 @@ if gidocgen_dep.found() and docs_python_deps.allowed() and gidocgen_app.found()
  install_data(['index.html', 'hsi.html'],
     install_dir : join_paths(datadir, 'doc', 'fwupd')
   )
+ install_data(['index.html', 'bios-settings.html'],
+    install_dir : join_paths(datadir, 'doc', 'fwupd')
+  )
  install_data(['urlmap_fwupd.js'],
     install_dir: join_paths(datadir, 'doc', 'libfwupd')
   )
  install_data(['urlmap_fwupdplugin.js'],
     install_dir: join_paths(datadir, 'doc', 'libfwupdplugin')
   )
+ install_symlink('libfwupd',
+    install_dir: join_paths(datadir, 'doc', 'fwupd'),
+    pointing_to: join_paths('..', 'libfwupd')
+ )
+ install_symlink('libfwupdplugin',
+    install_dir: join_paths(datadir, 'doc', 'fwupd'),
+    pointing_to: join_paths('..', 'libfwupdplugin')
+ )
```

Comment:
I don't get it...?

Reply:
When you install into the filesystem, neither of these are put in the right place relative to our index pages.  They are in the right place for what we do with Circle and fwupd.github.io though.  This makes sure both work.

Reply:
Ahh, and we need to use libfwupdplugin for devhelp to work.

Reply:
Oh I think I might get what's going on.  You might have broken doc deployment with 06a2dcf62b.

Reply:
> Ahh, and we need to use libfwupdplugin for devhelp to work.

Wait, so you're saying the directory needs to be `/usr/share/doc/libfwupdplugin` for devhelp to work?  If that can take a symlink I would rather we do it as the way I committed and then add a symlink.  Otherwise our doc deployment will still be broken too it seems to me because it expect we get everything from `DESTDIR/doc/fwupd/`

Reply:
Ahh, no, I'm wrong: /usr/share/doc/fwupd/libfwupdplugin/ is what's needed

Reply:
Okay so then the PR as it is now should work then.

Reply:
Nope, I'm double wrong, it has to be /usr/share/doc/libfwupd for it to work with devhelp -- i.e. no parent dir.

Reply:
Can you see if a symlink to fwupd/libfwupd works then for devhlp.

Reply:
Yes, that works. I'd be fine with the meson bump in this PR if that helps.

Reply:
Can we just pull that other PR then too perhaps?

Reply:
I have to sleep, but you have my blessing :)